core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -7578,6 +7578,21 @@ private void checkLiteral2(String expression, String expected) {
     sql(query).withPresto().ok(expected);
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6892">[CALCITE-6892]
+   *  CHAR_LENGTH Function is not recognized in DerbySQL.</a>. */
```

Comment:
It should be `CHAR_LENGTH Function is not recognized in DerbySQL`. We don't end with '.';

Reply:
Very thanks~ done @NobiGo